### PR TITLE
RDM-6390 functional test for revoke access to a case

### PR DESCRIPTION
**Pull request by RebeccaBaker**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/RDM-6390

### Change description ###
Functional tests for revoke access to a case F-046

**_NOTE: Will require re work when we have changes for the DSL_**

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ ] No
```

### CODE REVIEW

File: src/aat/resources/features/F-046/S-222.td.json [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+      "uid": "[[DYNAMIC]]",
+      "jid": "AUTOTEST1",
+      "ctid": "AAT",
+      "cid": "[[DYNAMIC]]",
+      "idToDelete": "[[DYNAMIC]]"
+    }
+  },
+  "expectedResponse": {
+    "responseCode": 204,
+    "responseMessage": "No content",
```
Comment (MSancaktutar) [inline src/aat/resources/features/F-046/S-222.td.json]: use  inheritance for 204 response

File: src/aat/resources/features/F-046/S-223.td.json [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+      "uid": "[[DYNAMIC]]",
+      "jid": "AUTOTEST1",
+      "ctid": "AAT",
+      "cid": "1234567890123456",
+      "idToDelete": "[[DYNAMIC]]"
+    }
+  },
+  "expectedResponse": {
+    "responseCode": 404,
+    "responseMessage": "Not Found",
```
Comment (MSancaktutar) [inline src/aat/resources/features/F-046/S-223.td.json]: use  inheritance for 404 response

File: src/aat/resources/features/F-046/S-223.td.json [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+    "headers": {
+      "Content-Length": "[[ANY]]",
+      "Content-Encoding" : "gzip",
+      "Content-Type" : "application/json;charset=UTF-8"
+    },
+    "body": {
+      "exception" : "uk.gov.hmcts.ccd.domain.service.getcase.CaseNotFoundException",
+      "timestamp": "[[ANY]]",
+      "status": 404,
+      "error": "Not Found",
```
Comment (MSancaktutar) [inline src/aat/resources/features/F-046/S-223.td.json]: use  inheritance for 404 response

**Comment by shrivsksavit** [comment]
continuous-integration/jenkins/pr-merge fail due to other test

**Pull request merged**